.travis.yml

```diff
@@ -5,7 +5,10 @@ python:
 install:
   - pip install pandas
   - pip install numpy
+  - pip install codecov
   # The latest version of doit supporting Python 2 is 0.29.0
   - test $(python -c 'import sys;print(sys.version_info[0])') -eq 2 && pip install 'doit==0.29.0' || pip install doit
 script:
-  - doit test
\ No newline at end of file
+  - doit test
+after_success:
+  - codecov
```

Comment:
For my understanding: what does this do, what the `coverage` call in `dodo.py` hasn't already done?

Reply:
This uploads the result of the tests that are run with coverage measurement enabled to the codecov.io servers.